ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/ExecutionPayloadUtil.java

```diff
@@ -15,7 +15,9 @@ public ExecutionPayloadUtil(ExecutionEngineService executionEngineService) {
     this.executionEngineService = executionEngineService;
   }
 
-  public ExecutionPayloadUtil() {}
+  public ExecutionPayloadUtil() {
+    this(ExecutionEngineService.createStub());
```

Comment:
I was thinking about this way of doing things but decided that explicit NPE (if no engine service is set) would be better than implicit misbehaviour. Nevertheless, I am completely OK with this change if you need this behaviour somewhere.

Reply:
Yeah, let it be that way for now. I think that both cases would need to refactored in production. I mean that calling `setExecutionEngineService` later doesn't look too safe to me as well